Library/Homebrew/cmd/test-bot.rb

```diff
@@ -427,7 +427,7 @@ def formula(formula_name)
         "#{@tap}/#{formula_name}"
       end
 
-      test "brew", "uses", "--recursive", canonical_formula_name
+      test "brew", "uses", canonical_formula_name
```

Comment:
If we're not checking recursive dependencies: how will we know if they are in need of a `revision` bump? They won't be checked by the missing deps if they aren't being read as `dependents`.